src/main/java/frc/robot/subsystems/swerve/Module.java

```diff
@@ -131,47 +133,45 @@ public void stop() {
     io.setDriveVoltage(0.0);
   }
 
-  /** Returns the current turn angle of the module. */
+  /** Returns the current turn angle of the module without PhoenixOdometryThread. */
```

Comment:
Might make more sense to say where this data comes from? As it's written I'm not actually sure.

Reply:
I think it's just the not high freq samples